integreat_cms/cms/utils/linkcheck_utils.py

```diff
@@ -41,13 +47,37 @@ def get_urls(
         # If the results should be limited to specific ids, filter the queryset
         urls = urls.filter(id__in=url_ids)
     if region_slug:
+        region = Region.objects.get(slug=region_slug)
+        latest_pagetranslation_versions = list(
+            PageTranslation.objects.filter(
+                page__id__in=Subquery(region.non_archived_pages.values("pk")),
+            )
+            .distinct("page__id", "language__id")
+            .values_list("pk", flat=True)
+        )
+        latest_poitranslation_versions = list(
+            POITranslation.objects.filter(poi__region=region)
+            .distinct("poi__id", "language__id")
+            .values_list("pk", flat=True)
+        )
+        latest_eventtranslation_versions = list(
+            EventTranslation.objects.filter(event__region=region)
+            .distinct("event__id", "language__id")
+            .values_list("pk", flat=True)
+        )
+        latest_imprinttranslation_versions = list(
```

Comment:
```suggestion
        latest_pagetranslation_versions = Subquery(
            PageTranslation.objects.filter(
                page__id__in=Subquery(region.non_archived_pages.values("pk")),
            )
            .distinct("page__id", "language__id")
            .values_list("pk", flat=True)
        )
        latest_poitranslation_versions = Subquery(
            POITranslation.objects.filter(poi__region=region)
            .distinct("poi__id", "language__id")
            .values_list("pk", flat=True)
        )
        latest_eventtranslation_versions = Subquery(
            EventTranslation.objects.filter(event__region=region)
            .distinct("event__id", "language__id")
            .values_list("pk", flat=True)
        )
        latest_imprinttranslation_versions = Subquery(
```
Here I guess we could also use subqueries instead of plain lists.